qgis4/ansible/roles/certification/tasks/main.yml

```diff
@@ -1,3 +1,7 @@
+- name: Import Nginx role
+  include_role:
```

Comment:
I am not fully sure but using `include_role` in each depending role won't the nginx role be executed multiple times when running the full playbook? I wonder if relying on [role dependencies](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-dependencies) wouldn't be a better option for that matter.

Reply:
It seems so indeed. Using dependencies instead leads to have nginx apt step executed only once. Checking 2 or 3 more things, and I'll update.